test/interactions/keyInteractionTests.ts

```diff
@@ -1,302 +1,340 @@
 ///<reference path="../testReference.ts" />
 
 describe("Interactions", () => {
-  describe("KeyInteraction", () => {
-    let svg: d3.Selection<void>;
-    let component: Plottable.Component;
-    let keyInteraction: Plottable.Interactions.Key;
-    let aCode: number;
-    let bCode: number;
-
-    let aCallbackCalled: boolean;
-    let bCallbackCalled: boolean;
-    let aCallback: Plottable.KeyCallback;
-    let bCallback: Plottable.KeyCallback;
-
-    beforeEach(() => {
-      aCode = 65;
-      bCode = 66;
-      component = new Plottable.Component();
-      svg = TestMethods.generateSVG(400, 400);
-      keyInteraction = new Plottable.Interactions.Key();
-      aCallback = () => aCallbackCalled = true;
-      bCallback = () => bCallbackCalled = true;
-      component.renderTo(svg);
-      aCallbackCalled = false;
-      bCallbackCalled = false;
-    });
-
+  describe("Key Interaction", () => {
     describe("onKeyPress", () => {
+      let svg: d3.Selection<void>;
+      let component: Plottable.Component;
+      let keyInteraction: Plottable.Interactions.Key;
+
+      let aKeyCode: number;
```

Comment:
Technically this should be `const`ed, or we can even capitalize case it.  What do you think?